roles/openshift_metrics/tasks/install_metrics.yaml

```diff
@@ -7,9 +7,10 @@
 
 - debug: msg="Created temp dir {{mktemp.stdout}}"
 
-- name: Create temp directory for all our templates
-  file: path={{mktemp.stdout}}/templates state=directory mode=0755
-  changed_when: False
+- name: copy `templates` directory
```

Comment:
In logging...i think this task is equivalent to 'generate all my objects that i will create later'  Are we certain we need this?